tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -393,9 +396,336 @@ var _ = ginkgo.Describe("[vol-allocation] Policy driven volume space allocation
 		// TODO: Verify no orphan volumes are created
 	})
 
+	/*
+		Verify EZT online volume expansion to a large size (should take >vpxd task timeout)
+		Steps:
+		1	Create a SPBM policy with EZT volume allocation
+		2	Create a SC using policy created in step 1 and allowVolumeExpansion set to true
+		3	Create a 2g PVC using SC created in step 2, say pvc1
```

Comment:
Please correct the steps according to what we are actually doing in tests

Reply:
steps are correct here, one of the logging messages in the test was incorrect, fixed it, thanks :)